native/cocos/bindings/manual/jsb_global.h

```diff
@@ -76,3 +77,8 @@ bool jsb_run_script(const ccstd::string &filePath, se::Value *rval = nullptr);
 bool jsb_run_script_module(const ccstd::string &filePath, se::Value *rval = nullptr); // NOLINT(readability-identifier-naming)
 
 bool jsb_global_load_image(const ccstd::string &path, const se::Value &callbackVal); // NOLINT(readability-identifier-naming)
+
+/**
+ * get the blocking timeout in milliseconds
+ */
+int32_t cc_get_blocking_timeout();     // NOLINT(readability-identifier-naming)
```

Comment:
Other functions starts as `jsb`, why not obey the rule?

Reply:
The blockingTimeout field, in reality, has no connection with JSB, yet I haven't identified a more suitable location for its storage. Any recommendations?

Reply:
The value is only used in WatchDog, so it is better to do everything in global. May be can use in ts like:
```
let watchDog = new native.WathDog();
watchDog.callback = xxx;
...
```

Reply:
When developers create the native.WatchDog object, they're also required to invoke the corresponding destroy method, which could be a significant imposition. Moreover, with respect to JS code, it's not possible to monitor all methods within a single tick in the same manner as in our current implementation. This discrepancy is tied to how setTimeout/requestAnimationFrame functions are implemented.

Reply:
```js
native.blockingDetection.timeout = 100;
native.blockingDetection.callback = (jsStack)=>{...}
```
Would these modifications potentially negate the need to make changes to the global namespace?